### DISCUSSION TITLE
OpenVPN API support

### DISCUSSION THREAD

**Discussion by yurtesen** [O-X-L/ansible-opnsense#32] [Q&A] [answered]
Hi, 
Do any work for supporting OpenVPN API started? Is there an estimate on when the OpenVPN API support would be ready?
Thanks!

**Comment by ansibleguy** [comment, marked as answer]
No, I haven't found time to implement it yet.. :/
Hope I can implement it until the end of this year

Reference: https://github.com/ansibleguy/collection_opnsense/issues/30

**Reply by yurtesen** [reply]
@ansibleguy thanks for the response. I found out that the `openvpn` API of OPNsense is not very useful since there is no way to upload certificates using API. We will probably continue using SSH/PHP for now. I hope this changes in future.
